### DISCUSSION TITLE
Thoughts on PWA before general 1.1.0 release

### DISCUSSION THREAD

**Discussion by cioraneanu** [cioraneanu/firefly-pico#49] [General]
In an attempt to make Pico feel more like a mobile app a basic PWA config was added. You can give it a try by checking out the `dev` tag from docker hub.

Please share feedback about your experience with it, so we can determine if we keep it, improve it or disable it until everything is sorted out.

I've seen 2 issues so far:
-  the initial cold start is way slower
- the top and bottoms toolbar sometime stay sticky sometimes not. 
![IMG_4586](https://github.com/cioraneanu/firefly-pico/assets/3276180/f769b0a3-c286-4bda-be74-071da4eb7b46)

**Comment by cioraneanu** [comment]
@avee87 Are you also experiencing slow starts when opening the PWA from homescreen / opening in browser incoginto?

**Reply by avee87** [reply]
Not really, it's pretty fast for me - both when opening from home screen and when opening in desktop browser.
I also haven't seen toolbar not being sticky.

Using Chrome on desktop and iOS/Safari for mobile

**Comment by byimam2nd** [comment]
Excuse me, may I ask, can firefly pico be run on vercel? I have tried running it there using the front/ folder but got an error:  

- Error: Request failed with status code 404
 
And error : 

- The provided endpoint is not correct.

When I try to add the access token in it.


I have created the environment FIREFLY_URL=https://firefly.domain and TZ before I run it.
